### DISCUSSION TITLE
ModuleNotFoundError: No module named 'jupyter_nbextensions_configurator'

### DISCUSSION THREAD

**Discussion by dwiretnoyanti00** [teamalgoritma/community#12] [Q&A] [answered]
Halo, saya sudah menginstall jupyter nbextensions di bagian base ..tapi pada saat dijalankan tab nbextension nya tidak muncul..saya keliru dimana ya ?Terima kasih bantuannya 🙏

**Comment by kevbow** [comment, marked as answer]
Halo Bu Dwi,

Ibu bisa menggunakan satu baris syntax tambahan ketika melakukan instalasi nbextension:

```
pip install jupyter_contrib_nbextensions
jupyter contrib nbextension install --user
jupyter nbextension enable varInspector/main
```
Setelah itu akan keluar info "Vaildating: ok" dan seharusnya tab nbextension sudah muncul. Jangan lupa untuk melakukan restart jupyter notebook sebelum nbextension bisa digunakan.
Demikian dan terima kasih.
